From 85d0f0c1b24b13b280ee624b26edbe123ab82f54 Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Wed, 24 May 2023 10:16:10 +0200
Subject: [PATCH] dt-bindings: display: simple: Add Youritech ET050WV05

Document the Youritech vendor prefix.
Add Youritech ET050WV05 5" WVGA (800x480) TFT LCD panel.

Signed-off-by: Daniel Ammann <daniel.ammann@bytesatwork.ch>
---
 .../devicetree/bindings/display/panel/panel-simple.yaml         | 2 ++
 Documentation/devicetree/bindings/vendor-prefixes.yaml          | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/panel/panel-simple.yaml b/Documentation/devicetree/bindings/display/panel/panel-simple.yaml
index 0763367dce0a..8822e877d9b1 100644
--- a/Documentation/devicetree/bindings/display/panel/panel-simple.yaml
+++ b/Documentation/devicetree/bindings/display/panel/panel-simple.yaml
@@ -307,6 +307,8 @@ properties:
       - winstar,wf35ltiacd
         # Yes Optoelectronics YTC700TLAG-05-201C 7" TFT LCD panel
       - yes-optoelectronics,ytc700tlag-05-201c
+        # Youritech ET050WV05 5" WVGA (800x480) TFT LCD panel
+      - youritech,et050WV05
 
 if:
   not:
diff --git a/Documentation/devicetree/bindings/vendor-prefixes.yaml b/Documentation/devicetree/bindings/vendor-prefixes.yaml
index 685c570feb37..7078d3e4d318 100644
--- a/Documentation/devicetree/bindings/vendor-prefixes.yaml
+++ b/Documentation/devicetree/bindings/vendor-prefixes.yaml
@@ -1709,6 +1709,8 @@ patternProperties:
     description: YSH & ATIL
   "^yones-toptech,.*":
     description: Yones Toptech Co., Ltd.
+  "^youritech,.*":
+    description: Youritech
   "^ys,.*":
     description: Shenzhen Yashi Changhua Intelligent Technology Co., Ltd.
   "^ysoft,.*":
